apps/ledger-live-mobile/src/components/DeviceAction/InstallSetOfApps/index.tsx

```diff
@@ -80,7 +80,7 @@ const InstallSetOfApps = ({
           <Text mb={5} variant="paragraphLineHeight">
             {listingApps ? (
               <Trans i18nKey="installSetOfApps.ongoing.resolving" />
-            ) : progress || currentAppOp ? (
+            ) : progress && currentAppOp ? (
```

Comment:
possible change of logic here. also if progress===0 it's maybe legit to enter here (and display 0% with spinner)

Reply:
unsure about this one, the previous check was clearly wrong since progress could have been `undefined`
we'll need the relevant team to check this (@LedgerHQ/live-devices maybe since this is device action stuff?)

Reply:
i remember when developing this that there are cases where we don't have a progress YET we want to display a LOADER **without** a percentage yet (progress), so i'm pretty sure the `||` is intended here. maybe instead we want to `{ typeof progress === "number" ? <Trans thingy > : null }` and still let the "loader" happening

Reply:
will do 👍